### DISCUSSION TITLE
Rango de ID de pinguino

### DISCUSSION THREAD

**Discussion by aamenabarr** [IIC2133-PUC/2024-2#50] [Tarea 0] [answered]
Si nos dicen que la capacidad es 50, todos los IDs van a estar entre 0 y 50 o pueden haber ID más grandes que la capacidad?

**Comment by tomastrivino** [comment, marked as answer]
La capacidad de la tienda representa únicamente cuántos pingüinos pueden haber simultáneamente. No tiene relación al rango de los IDs de los pingüinos, sino cuántos pueden haber en un cierto instante dentro de esta como máximo.
